src/components/cg-displayer-details.test.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React from 'react';
+
+import { faker } from '@faker-js/faker';
+
+import { CGDisplayerDetails } from './cg-displayer-details';
+import { screen, setupTest } from '../carbonio-ui-commons/test/test-setup';
+import { TESTID_SELECTORS } from '../constants/tests';
+import { buildContactGroup, buildMembers } from '../tests/model-builder';
+
+describe('Contact group displayer details', () => {
+	it('should display the name of the CG', () => {
+		const title = faker.word.noun();
+		setupTest(<CGDisplayerDetails contactGroup={buildContactGroup({ title })} />);
+		expect(screen.getByText(title)).toBeVisible();
+	});
+
+	it('should displays the number of the members of the CG', () => {
```

Comment:
typo (also on the test after this one)
```suggestion
	it('should display the number of the members of the CG', () => {
```